lib/snowplow_tracker/emitter.ex

```diff
@@ -40,16 +40,19 @@ defmodule SnowplowTracker.Emitter do
         emitter.request_type
       )
 
-    with {:ok, response} <- Request.get(url, [], default_options()),
-         {:ok, body} <- Response.parse(response) do
-      {:ok, body}
-    else
-      {:error, error} ->
-        raise Errors.ApiError, Kernel.inspect(error)
-    end
+    Unary.create(payload, url)
```

Comment:
can you add an alias? - `UnaryEmitter`

Reply:
Since it's already in the emitter file, is this required?